test/image_test.py

```diff
@@ -750,6 +750,48 @@ def f(self):
         print("bar!", VARIABLE_6)
 
 
+class FooInstance:
+    bar: str = "5"
+    barbar: str = "2"
+
+    @build()
+    def build_func(self):
+        global VARIABLE_5
+
+        print("foo!", VARIABLE_5)
+        print("static var", FooInstance.bar)
+
+
+def test_image_cls_var_rebuild(client, servicer):
+    rebuild_app = App()
+    image_id = -1
+    rebuild_app.cls(image=Image.debian_slim())(FooInstance)
+    with rebuild_app.run(client=client):
+        image_id = list(servicer.images.keys())[-1]
```

Comment:
Not worth rewriting the code, but just an FYI, you can just do `list(my_dict)` to get a list of the keys, you don't need to call `.keys()` directly.

Reply:
Done